recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -0,0 +1,38 @@
+FIRMWARECONTROL_PLATFORM ?= "none"
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON \
+', '', d)}"
+
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'broadcom', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_BCM=ON \
+', '', d)}"
+
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'arris', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_BCM=OFF \
+', '', d)}"
+
+# Set dependencies
+WPE_FIRMWARECONTROL_DEP ?= "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi', 'mfrlibs-rpi', '', d)}"
+WPE_FIRMWARECONTROL_DEP ?= "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'broadcom', 'broadcom-mfrlib', '', d)}"
+WPE_FIRMWARECONTROL_DEP ?= "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'arris', 'flashapp-lib', '', d)}"
+
+DEPENDS += " ${WPE_FIRMWARECONTROL_DEP} "
```

Comment:
Does your dependencies must be available in the runtime or only in the build time? If yes then please add RDEPENDS.